Sources/Benchmark/BenchmarkMetric.swift

```diff
@@ -193,12 +195,14 @@ public extension BenchmarkMetric {
             return "Bytes (read physical)"
         case .writeBytesPhysical:
             return "Bytes (write physical)"
+        case .objectAllocCount:
+            return "Object allocs"
         case .retainCount:
             return "Retains"
         case .releaseCount:
             return "Releases"
         case .retainReleaseDelta:
-            return "Retain / Release Δ"
+            return "(Alloc + Retain) - Release Δ"
```

Comment:
```suggestion
            return "(Alloc + Retain) / Release Δ"
```
otherwise it looks like delta of difference?

Reply:
It is the difference delta that is being display :-)